python/ray/serve/tests/test_api.py

```diff
@@ -646,6 +646,31 @@ def __call__(self):
     assert requests.get("http://127.0.0.1:8000/").text == "got model"
 
 
+@pytest.mark.parametrize(
+    "ingress_route,app_route",
+    [("/hello", "/"), ("/hello", "/override"), ("/", "/override")],
```

Comment:
Could you also add tests for (`None`, `None`), (non-`None`,`None`) and (`None`, non-`None`)?

Reply:
Good point, will do!

Reply:
Added!